### DISCUSSION TITLE
Basic Idea and Features

### DISCUSSION THREAD

**Discussion by hrahul2605** [dsc-iiest/community-leaderboard#2] [Ideas]
Current Idea: A community leaderboard to track fitness of the members in the community.

- Should we make the project more generic? Instead of only focusing on fitness tracking, we can have other domains as well
- What features should we enable ?
- Technology to be used?

**Comment by hrahul2605** [comment]
Features:
- Longest streak
- Follow friends
- Maintain global leaderboard & only friends

**Comment by Sanjana00** [comment]
Input: have you met your fitness goals today (a switch that can be set as
True only once in 24 hours)

Optional: amount of water consumed today, medicines taken (yes/no) - only
for personal journaling purposes, does not show up on the leaderboard
